### ISSUE TITLE
Build failure for d47b18ead07f99181db87d2

### ISSUE THREAD

**Issue by chdiza** [macvim-dev/macvim#472]
I just pulled d47b18ead07f99181db87d2, and I can't build anymore:

```
gcc -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_MACVIM -Wall -Wno-unknown-pragmas -pipe  -DMACOS_X_UNIX  -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        -o objects/misc1.o misc1.c
menu.c:2661:20: warning: implicit declaration of function 'vim_iswhite' is
      invalid in C99 [-Wimplicit-function-declaration]
        while (*linep && !vim_iswhite(*linep) && *linep != '=')
                          ^
1 warning generated.
```

Later on:

```
Undefined symbols for architecture x86_64:
  "_vim_iswhite", referenced from:
      _ex_macmenu in menu.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [Vim] Error 1
make: *** [first] Error 2
```

I think there must have been a merge error of some kind, because I don't get this in regular Vim, and I did not get it in 824b7fa3c30ac3b5c.

My build flags:
```
--disable-netbeans
--disable-nls
```

**Comment by chdiza** [comment]
Probably that `vim_iswhite` should be all caps because of recent Vim commits.

**Comment by chdiza** [comment]
Yep, that was it.  If I manually upcase it, build proceeds fine.  Must've just been missed in the merge.  I dunno how CI didn't catch this.

**Comment by splhack** [comment]
Thanks, fixed in cc99aa38bd249282c84c8bc0105ea6c9dc6e2423
